.circleci/config.yml

```diff
@@ -230,7 +230,7 @@ jobs:
             # points to a broken IP
             echo "127.0.0.1 metriton.datawire.io" | sudo tee -a /etc/hosts
             DTEST_KUBECONFIG="$HOME/kubeconfig" PATH="/c/:/c/Program Files/SSHFS-Win/bin:$PATH" DTEST_REGISTRY="docker.io/datawire" make test
-          no_output_timeout: 27m
+          no_output_timeout: &testsTimeout 27m
```

Comment:
I had no idea you could do this in circle.

Reply:
it's not so much a circle thing as just a [yaml feature](http://blogs.perl.org/users/tinita/2019/05/reusing-data-with-yaml-anchors-aliases-and-merge-keys.html)